src/features/auth/model/useAuth.ts

```diff
@@ -0,0 +1,56 @@
+import { useNavigate } from 'react-router-dom';
+import { useStore } from '@shared/lib/useStore';
+import type { LoginOptions } from '../login/interfaces/login-options.interface';
+import type { RegistrationOptions } from '../registaion/interfaces/registaion-options.interface';
+import { loginApi } from '../login/api/login.api';
+import { registrationApi } from '../registaion/api/registration.api';
+import type { AuthResponse } from '../interfaces/auth-response.interface';
+import { setCookies } from './setCookies';
+
+interface UseAuthReturnType {
```

Comment:
```suggestion
interface AuthReturnType {
```
Only hooks started from 'use'